modules/storage/config.go

```diff
@@ -62,6 +63,10 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	cfg.Trace.Local = &local.Config{}
 	f.StringVar(&cfg.Trace.Local.Path, util.PrefixConfig(prefix, "trace.local.path"), "", "path to store traces at.")
 
+	cfg.Trace.BackgroundCache = &cortex_cache.BackgroundConfig{}
+	f.IntVar(&cfg.Trace.BackgroundCache.WriteBackBuffer, util.PrefixConfig(prefix, "trace.background_cache.writeback_buffer"), 10000, "Key batches to buffer for background write-back.")
+	f.IntVar(&cfg.Trace.BackgroundCache.WriteBackGoroutines, util.PrefixConfig(prefix, "trace.background_cache.writeback_goroutines"), 10, "Concurrency to write back to cache.")
```

Comment:
Do we need flags for this cache config? I think it is advanced enough to go into the config _only_, but will let you decide. (Also if we decide to go for flags, the underscores `_` need to be replaced with hyphens `-`)

Reply:
Probably, we don't. Agree! Removed the flags.